ursula_cli/shell.py

```diff
@@ -195,7 +195,8 @@ def _run_heat(args, hot):
     CREDS = {
         'username': os.environ['OS_USERNAME'],
         'password': os.environ['OS_PASSWORD'],
-        'tenant_name': os.environ['OS_TENANT_NAME'],
+        'tenant_name': (os.environ.get('OS_TENANT_NAME') or
+                        os.environ['OS_PROJECT_NAME']),
```

Comment:
Patch away. I'm happy to see any work to make this block more resilient. While you're at it, you could check for OS_TENANT_ID or OS_PROJECT_ID too, since those might be used instead of the name.